app/src/pages/prompt/PromptLatestVersionsList.tsx

```diff
@@ -42,42 +44,30 @@ export function PromptLatestVersionsList(props: {
   }
 
   return (
-    <ul>
-      {versions.map((version) => {
-        return (
-          <li key={version.id} css={versionListItemCSS}>
-            <Flex direction="row" gap="size-200" alignItems="start">
-              <Icon svg={<Icons.Commit />} />
-              <Flex direction="column" width="100%" gap="size-50">
-                <View width="100%">
-                  <Flex
-                    direction="row"
-                    justifyContent="space-between"
-                    alignItems="center"
-                  >
-                    <span>{version.id}</span>
-                    <Text color="text-300">
-                      {formatRelative(version.createdAt, Date.now())}
-                    </Text>
-                  </Flex>
-                </View>
-                <Text color="text-700">{version.description}</Text>
-              </Flex>
-            </Flex>
-            {/* TODO(prompts): show that there are more */}
-            <VersionsConnector />
-          </li>
-        );
-      })}
-    </ul>
+    <div>
+      <ul>
+        {versions.map((version) => {
+          return (
+            <li key={version.id} css={versionListItemCSS}>
+              <PromptVersionSummary promptVersion={version} />
+              {/* TODO(prompts): show that there are more */}
+              {version.sequenceNumber != 1 ? <VersionsConnector /> : null}
+            </li>
+          );
+        })}
+      </ul>
```

Comment:
Can we put some kind of element here when version number is not 1 but we're at the last index?

Something to cap the line but signify more items. 

Or we can maybe reduce the opacity of the last line when there are more to load

Reply:
done

Reply:
<img width="464" alt="Screenshot 2025-01-16 at 9 57 05 PM" src="https://github.com/user-attachments/assets/5f9f7db9-ae02-47bb-bb2c-ee06a066cdc8" />